test/export-integration.test.js

```diff
@@ -0,0 +1,108 @@
+/*
+This file contains integration tests for the exportDataset function and are based on fixtures
+  in the fixtures directory. Each fixture contains a set of test cases that are run against the
+  exportDataset function and a mocked backend API with disabled network requests.
+*/
+
+const fs = require('fs')
+const yaml = require('yaml')
+const nock = require('nock')
+const path = require('path')
+const {afterAll, describe, expect, test} = require('@jest/globals')
+const exportDataset = require('../src/export')
+const rimraf = require('../src/util/rimraf')
+const sanity = require('@sanity/client')
+const {untarExportedFile, ndjsonToArray} = require('./helpers')
+const {newTestRunId, withTmpDir} = require('./helpers/suite')
+
+const fixturesDirectory = path.join(__dirname, 'fixtures')
+const testif = (condition) => (condition ? test : test.skip)
+
+const assertExportSuccess = async (exportDir, exportFilePath, dataContent, assetsContent) => {
+  expect(fs.existsSync(exportFilePath)).toBeTruthy()
+  const stats = fs.statSync(exportFilePath)
+  expect(stats.size).toBeGreaterThan(0)
+
+  const extractedDir = await untarExportedFile(exportDir, exportFilePath)
+  expect(fs.existsSync(extractedDir)).toBeTruthy()
+  expect(ndjsonToArray(fs.readFileSync(`${extractedDir}/data.ndjson`, 'utf8'))).toEqual(dataContent)
+  expect(JSON.parse(fs.readFileSync(`${extractedDir}/assets.json`, 'utf8'))).toEqual(assetsContent)
```

Comment:
Could use a snapshot instead? It's more idiomatic to jest/node tests. They are also easy to update with `-u`

```suggestion
  expect(ndjsonToArray(fs.readFileSync(`${extractedDir}/data.ndjson`, 'utf8'))).toMatchSnapshot()
  expect(JSON.parse(fs.readFileSync(`${extractedDir}/assets.json`, 'utf8'))).toMatchSnapshot()
```

Reply:
Good point. Fixed it.